### ISSUE TITLE
False Positives for async-http-client

### ISSUE THREAD

**Issue by ramsperger** [dependency-check/DependencyCheck#1068]
async-http-client includes dependencies with "netty" in the name resulting in false positives for old versions of netty.

All reported as cpe:/a:netty_project:netty:2.0.38

- async-http-client-netty-utils-2.0.38.jar
- netty-resolver-2.0.38.jar
- netty-codec-dns-2.0.38.jar
- netty-reactive-streams-1.0.8.jar

For dependency:
```xml
<dependency>
    <groupId>org.asynchttpclient</groupId>
    <artifactId>async-http-client</artifactId>
    <version>2.0.38</version>
</dependency>
```

**Comment by jeremylong** [comment]
Thanks - yeah, I've seen a lot of FP around libraries used with netty being reported as netty itself. I'm still trying to come up with a better way of handling these.  I will include this in the next pass of FP suppression rules.

Thanks!
